frontend/src/components/ModalContent.vue

```diff
@@ -1,119 +1,116 @@
 <template>
   <v-card>
-    <v-card-title class="title-header">
-      <v-row>
-        <p class="modal-title">{{ selectedProject.name }}</p>
-        <v-card-actions>
-          <v-btn
-            class="mr-2"
-            absolute
-            fab
-            rounded
-            icon
-            large
-            right
-            @click="onClick()"
-          >
-            <v-icon size="3.5rem" color="white">close</v-icon>
-          </v-btn>
-        </v-card-actions>
-      </v-row>
+    <v-card-title class="container-title unified-padding">
+      <h1>{{ selectedProject.name }}</h1>
+      <div>
+        <v-btn class="hidden-xs-only" rounded icon large @click="onClick()">
+          <v-icon size="3.5rem" color="white">close</v-icon>
+        </v-btn>
+      </div>
     </v-card-title>
-    <v-card-actions>
-      <div class="buttons-list">
-        <v-row>
-          <v-btn
-            class="buttons"
-            text
-            rounded
-            large
-            :href="selectedProject.licensePage"
-            target="_blank"
-          >
-            <v-icon>far fa-copyright</v-icon>
-            <p class="buttons--text">
-              Licencja {{ selectedProject.licenseName }}
-            </p>
-          </v-btn>
-          <v-btn
-            class="buttons"
-            text
-            rounded
-            large
-            :href="selectedProject.githubLink"
-            target="_blank"
-          >
-            <v-icon>fab fa-github</v-icon>
-            <p class="buttons--text">Repozytorium</p>
-          </v-btn>
-          <v-btn
-            class="buttons"
-            text
-            rounded
-            large
-            :href="selectedProject.websiteLink"
-            target="_blank"
+    <v-card-actions class="buttons-list unified-padding">
+      <div v-show="selectedProject.licensePage !== ''">
+        <v-btn text rounded :href="selectedProject.licensePage" target="_blank">
+          <v-icon>far fa-copyright</v-icon>
```

Comment:
why not to use https://vuetifyjs.com/en/features/icons/#using-custom-icons  and create component with icons? this way you write it once and then use it in a whole project
also I'd add some accessibility here like `aria-label`

Reply:
That seems awsome. I'll try to implement that.

Reply:
**Final changes:**
Global store for icons seems to be a really good idea but when I started playing with it both through Vuetify and Font Awsome I couldn't find the way that actually save code. To the contrary, it did introduce another layer of abstraction which doesn't seem that necessary at this point. For now I settled with the basic local use of FA with <i> tag. 

I still believe global store for icons is a good idea but I think it goes beyond the scope of this task to do it right. I will create a new issue for that.